e2e/testdata/fn-render/missing-kptfile/.expected/config.yaml

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 exitCode: 1
-stdErr: "missing-kptfile/Kptfile: no such file or directory"
\ No newline at end of file
+stdErr: "No Kptfile found at"
```

Comment:
incomplete message

Reply:
This is intentional. The check does a substring search and the error message includes the path to the package, which is not possible to predict before the test runs. We could resolve the full path when the test runs and put it into the expected message when the test runs, but not sure if that is worth the complexity.